src/libraries/System.Diagnostics.EventLog/src/System.Diagnostics.EventLog.csproj

```diff
@@ -3,6 +3,10 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent);$(NetCoreAppMinimum)-windows;$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
     <NoWarn>$(NoWarn);CA1838;CA1847</NoWarn>
+    <!-- Suppressions to avoid ifdefs:
+            CA1845: Use span-based 'string.Concat' and 'AsSpan' instead of 'Substring'
+            CA1846: Prefer 'AsSpan' over 'Substring' when span-based overloads are available -->
+    <NoWarn>$(NoWarn);CA1845;CA1846</NoWarn>
```

Comment:
What's causing these analyzers to fire now and they didn't before?

Reply:
The reference to `System.Memory` (used by some parts of the generated p/invoke stubs).

Reply:
I see; then the analyzer sees that these APIs exist and the analyzers start running.  Ok, thanks.